smoke-tests/apps/DiagnosticExtension/src/smokeTest/resources/applicationinsights.json

```
@@ -0,0 +1,14 @@
+{
+  "role" : {
+    "name" : "testrolename",
+    "instance" : "testroleinstance"
+  },
+  "preview" : {
```

technically this smoke test doesn't need this since it's not verifying telemetry, but just to stay consistent with the others:
```suggestion
  "sampling": {
    "percentage": 100
  },
  "preview" : {
```